packages/relay/src/lib/eth.ts

```diff
@@ -277,6 +284,19 @@ export class EthImpl implements Eth {
     this.common = new CommonService(mirrorNodeClient, logger, cacheService);
     this.filterServiceImpl = new FilterService(mirrorNodeClient, logger, cacheService, this.common);
     this.debugServiceImpl = new DebugService(mirrorNodeClient, logger, this.common);
+    this.emptyTrieRoot = EthImpl.emptyHex;
```

Comment:
If we do this we can get rid of the `getEmptyTrieRoot()` function and directly use `EthImpl.emptyTrieRoot`:
```suggestion
    this.emptyTrieRoot = `0x${toHexString(new Trie({}))}`;
```

Reply:
I checked the source code of `ethereumjs` and it seems that the reason this `create` method is `async` is to await the connection to the DB, if any is passed through the arguments, e.g., `await Trie.create({ db: new MapDB() })`. 

But in this case, we want to use only the `EMPTY_TRIE_ROOT` constant that is inside of it, so we have no reason to call `Trie.create({})` and overcomplicate the code with this asynchronous method, instead of just initializing it through the constructor `new Trie()`.

Reply:
If no arguments are passed, there should be no difference between doing `await Trie.create()` and `new Trie()`